pysiaf/aperture.py

```diff
@@ -303,7 +303,7 @@ def complement(self):
             setattr(self, 'XIdlVert{:d}'.format(j), corners_Idl_x[j - 1])
             setattr(self, 'YIdlVert{:d}'.format(j), corners_Idl_y[j - 1])
 
-    def convert(self, X, Y, from_frame, to_frame):
+    def convert(self, X, Y, from_frame, to_frame) -> object:
```

Comment:
Let's get rid of the check failures first. Please remove the `-> object` text (why is it here?). Then commit and push to the branch. That should allow the tests to be run.